eoscostarica/guide.eoscostarica.io
# Migrate docusaurus2

### Migrate to docusaurus v2 

#83 

### Steps to test
1. Run project locally
1. See site and review content

#### CheckList
